content/post/cloud-init-cloud-on-hypriot-x64.md

```diff
@@ -0,0 +1,348 @@
++++
+Categories = ["Docker", "Raspberry Pi", "ARM", "ARM64"]
+Tags = ["Docker", "Raspberry Pi", "ARM", "ARM64", "External" "cloud-init", "rpi-64"]
+Description = "Bootstrapping a Cloud with Cloud-Init and HypriotOS"
+date = "2017-10-29T00:14:00-07:00"
+more_link = "yes"
+title = "Bootstrapping a Cloud with Cloud-Init and HypriotOS"
+disqus = "yes"
+social_sharing = "yes"
+
++++
+
+> Things may come to those who wait, but only the things left by those who hustle.
+
+Over the last year, a lot has happened in the Raspberry Pi and Docker communities, there are captains helping lead the charge, one of those, [Dieter Reuter](https://twitter.com/Quintus23M) really has been pushing the cause for ARM64 support with both Raspberry Pi 3 and LinuxKit. He isn't a single man army, the rest of the [Docker Pirates](http://blog.hypriot.com/crew/) over at [Hypriot](http://blog.hypriot.com/) has been doing some awesome things as well!
+
+Building on the backs of these outstanding community members, I was finally able to spin up a RaspberryPi, exactly like I do in the "real world", just turn it on, let it self-configure, then software just runs.
+
+<!--more-->
+
+A lot of this is really about having [cloud-init ](http://cloudinit.readthedocs.io/en/latest/index.html) available in the HypriotOS 64bit images thanks to another Pirate [Stefan Scherer](https://twitter.com/stefscherer). In the "cloud world", you spin up instances, give it some user-data, then when it boots, the machines self-configures based on the instance meta-data and the user-data provided.
+
+## Before
+
+Before we talk about the "new" pirate way, let't talk about the "old" non-pirate way.
+
+* Download Jessie Lite (the easiest for nubbies like me)
+* Flash said image to an SD Card
+* Optionally re-mount SD Card and monkey some stuff around
+* Put the card in your PI
```

Comment:
s/PI/RPi/
I think using the term `RPi` as the abbreviation for `Raspberry Pi` seems a little bit more consistent